.drone.yml

```diff
@@ -234,6 +234,37 @@ steps:
     event:
     - tag
 
+- name: docker-tag
+  image: rancher/dapper:v0.5.4
+  environment:
+    AWS_REGISTRY_ID:
+      from_secret: aws_registry_id
+  commands:
+  - "docker tag rancher/rancher:${DRONE_TAG} ${AWS_REGISTRY_ID}.dkr.ecr.us-east-1.amazonaws.com/suse/rancher:${DRONE_TAG}"
+  privileged: true
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+  when:
+    event:
+    - tag
+
+- name: docker-publish-ecr
+  image: plugins/ecr
+  environment:
+    AWS_REGISTRY_ID:
+      from_secret: aws_registry_id
+  settings:
+    access_key:
+      from_secret: ecr_access_key
+    secret_key:
+      from_secret: ecr_secret_key
+    repo: suse/rancher
+    registry: "${AWS_REGISTRY_ID}.dkr.ecr.us-east-1.amazonaws.com"
+  when:
+    event:
+    - tag
```

Comment:
We don't want to do this on every tag, just core ones. Meaning no pre-releases, rc's, etc..

Reply:
How can that be expressed by Drone events?

Reply:
Would this work?

```
  when:
    ref:
      include:
        - "refs/heads/release/v*"
    event:
      - tag
```

Reply:
No because we tag everything off the `release/v2.6` branch. The tag `v2.6.3` and `v2.6.3-rc1` are done the same way. You'd have to filter on the tag name.

Reply:
`refs/tags/v*` may work, though this will include `-rcX` tags. @cbron we may need RCs for testing initially, right?

Reply:
if we want to exclude `-rcX` tags, we can use something like:
```
include:
- "refs/tags/v*"
exclude:
- "refs/tags/*-*"
```
(from another rule in `.drone.yml`)

Reply:
I can add that, but how will we test it? Wait for the next official release?

Reply:
@cbron @ryansann any ideas?

Reply:
You could leave the excludes out to test an rc, then delete it, as long as you can delete images. Or just wait for final tag and if that doesn't work manually mirror.